### ISSUE TITLE
Plugin crashes on IDE startup.

### ISSUE THREAD

**Issue by R-Bower** [TeamCodeStream/CodeStream#95]
**Describe the bug**
Codestream Plugin fails to load on IntelliJ IDEA Ultimate.  The IDEA does not crash, but the plugin error appears with a tracktrace and it remains unusable.

**System Info (please complete the following information):**
 - Version of the CodeStream extension: 2.1.0+1
 - OS: 
    Linux Mint 19.1
    5.2.11-050211-generic
 - Intellij IDEA Ultimate 2019.2.1 - idea-IU-192.6262.58

**Log Files**
```
2019-09-05 13:07:29,809 [   3538]   INFO - .codestream.agent.AgentService - Initializing CodeStream LSP agent 
2019-09-05 13:07:30,071 [   3800]   INFO - j.ide.script.IdeStartupScripts - 0 startup script(s) found 
2019-09-05 13:07:30,152 [   3881]   INFO - .codestream.agent.AgentService - CodeStream LSP agent extracted to /tmp/codestream17722175128169212956.tmp/codestream-agent 
2019-09-05 13:07:30,152 [   3881]   INFO - .codestream.agent.AgentService - xdg-open extracted to /tmp/codestream17722175128169212956.tmp/xdg-open 
2019-09-05 13:07:30,230 [   3959]   INFO - .codestream.agent.AgentService - LSP agent terminated with exit code 4 
2019-09-05 13:07:30,231 [   3960]   WARN - .codestream.agent.AgentService - Pkg: FLAGS_MISMATCH 
2019-09-05 13:07:30,251 [   3980]  ERROR - .codestream.agent.AgentService - java.io.IOException: Stream closed 
org.eclipse.lsp4j.jsonrpc.JsonRpcException: java.io.IOException: Stream closed
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:72)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.request(RemoteEndpoint.java:161)
	at org.eclipse.lsp4j.jsonrpc.services.EndpointProxy.invoke(EndpointProxy.java:91)
	at com.sun.proxy.$Proxy82.initialize(Unknown Source)
	at com.codestream.agent.AgentService.initAgent(AgentService.kt:98)
	at com.codestream.agent.AgentService.initAgent$default(AgentService.kt:81)
	at com.codestream.agent.AgentService$1.invokeSuspend(AgentService.kt:71)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:238)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:742)
Caused by: java.io.IOException: Stream closed
	at java.base/java.lang.ProcessBuilder$NullOutputStream.write(ProcessBuilder.java:442)
	at java.base/java.io.OutputStream.write(OutputStream.java:157)
	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:69)
	... 11 more

```

**Comment by planteater** [comment]
@R-Bower - Thanks for the report! Can you let me know if you're running 32bit or 64bit Mint? Unfortunately, we only support 64bit Linux at the moment.

If you are running 64bit, could I trouble you to send a log file? Instructions for locating the log file can be found here:

https://github.com/TeamCodeStream/CodeStream/wiki/Instructions-for-finding-CodeStream-log-files

If you'd prefer, feel free to send the log file to me directly at dave@codestream.com. Thanks!

**Comment by R-Bower** [comment]
I'm on 64bit.

[idea-logs-20190905-162347.zip](https://github.com/TeamCodeStream/CodeStream/files/3581553/idea-logs-20190905-162347.zip)

**Comment by marcelofarias** [comment]
We are working on a fix. In the meanwhile, could you check if you have an environment variable called `NODE_OPTIONS` set? If so, could you try to clear it and restart IntelliJ?

**Comment by planteater** [comment]
@R-Bower - Fixed in v2.1.3
